messaging/src/main/java/org/axonframework/messaging/eventhandling/processing/streaming/token/store/jpa/JpaTokenStore.java

```diff
@@ -398,7 +428,7 @@ public CompletableFuture<List<Segment>> fetchAvailableSegments(@Nonnull String p
      * @throws UnableToClaimTokenException If there is a token for given {@code processorName} and {@code segment}, but
      *                                     it is claimed by another process.
      */
-    protected TokenEntry loadToken(String processorName, int segment, EntityManager entityManager) {
+    private TokenEntry loadToken(String processorName, int segment, EntityManager entityManager) {
```

Comment:
Made some methods private as otherwise I would need to add a `ensureStoreUsable` guard there as well.